spec/cohortManager.spec.js

```diff
@@ -0,0 +1,275 @@
+const CohortManager = require('../src/cohortManager.js')
+
+describe('Cohort Manager', () => {
+  let app
+
+  beforeEach(() => {
+    app = new CohortManager()
+  })
+
+  it('creates a new cohort', () => {
+    const expected = {
+      id: 1,
+      cohortName: 'cohort10',
+      students: []
+    }
+    const result = app.addCohort('cohort10')
+    expect(result).toEqual(expected)
+  })
+  it('try to add a new cohort without a name', () => {
+    const result = () => app.addCohort('')
```

Comment:
Could you explain why is this being called as a function? Would having res = app.addCohort(' ') work the same?

Reply:
This is related to the TrowError(). I used this as a reference https://stackoverflow.com/questions/4144686/how-can-i-write-a-test-which-expects-an-error-to-be-thrown-in-jasmine